src/file_worker.py

```diff
@@ -73,6 +72,9 @@ class DatasetIndexScopeType(Enum):
     PUBLIC = 'PUBLIC'
 
 class FileWorker:
+    # Constants to build endpoint URLs for Ontology API
+    ONTOLOGY_API_ORGAN_TYPES_ENDPOINT = '/organs/by-code?application_context=HUBMAP'
+    ONTOLOGY_API_ASSAY_TYPES_ENDPOINT = '/assaytype?application_context=HUBMAP'
```

Comment:
@kburke It's unclear to me that are you replacing the assaytype yaml with this UBKG ontology-api call too? I can still see `ASSAY_TYPES_YAML` being used at the same time. Can you confirm?

Reply:
@yuanzhou You are correct, this PR **_does not_** deal with the assay types YAML.  I've left the line in there which reads what the ontology-api offers for assay types, but integrating that as a replacement is work I planned on https://github.com/hubmapconsortium/files-api/issues/46.